scaleway/helpers_domain.go

```diff
@@ -30,8 +30,8 @@ func flattenDomainData(data string, recordType domain.RecordType) interface{} {
 		if len(dataSplit) == 2 {
 			return dataSplit[1]
 		}
-	default:
-		return data
+	case domain.RecordTypeTXT:
```

Comment:
Could we return directly in the switch clause? It seems easier to follow

Reply:
ok good 👍 idea